### ISSUE TITLE
ResNorm should have multiple outputs

### ISSUE THREAD

**Issue by ElliotAOram** [mantidproject/mantid#13717]
In order to avoid calls to the ADS in the ResNorm GUI. The ResNorm algorithm should be adapted to have multiple outputs rather than the current functionality of adding workspaces to the ADS. This may involve temporarily getting output workspaces from the ADS due to the nature of plot peaks by log value.

**Comment by ElliotAOram** [comment]
The algorithm currently deletes the input `_res` workspace which is referenced in #13713. These two issue can just be merged into this single issue as this issue will solve the other.

**Comment by ElliotAOram** [comment]
This issue is blocked by #13678. Many of the output workspaces are from PlotPeakByLogValue so it makes sense to handle this issue first.

**Comment by ElliotAOram** [comment]
Algorithm outputs WorkspaceGroup and TableWorkspace - other outputs are from `PlotPeaksByLogValue`. This issue has been resolved and can be closed
